sandbox/tests/block-builder_test.rs

```diff
@@ -28,17 +30,52 @@ fn test_block_builder_apply_extrinsics() {
 #[test]
 fn test_block_builder_inherent_extrinsics() {
     let mut setup = Setup::new();
-    let inh = Inherent::new();
+    let mut inh = InherentData::new();
```

Comment:
Maybe you can add new function in this file that returns the inherent data map, ready to be passed?
This way the test will be cleaner and the composition of the `inherent_data` will be separated